website/src/Website.vue

```diff
@@ -4,7 +4,10 @@
 		<section class="Website__content">
 
 			<p>
-				Welcome to Holistic Toolbox. We provide a growing list of tools to make working easier.
+				We're a team of developers striving to create a useful set of tools for developers, computer scientists and curious folk.
```

Comment:
Can you add a screenshot of what this looks like? As mentioned on the call I think potentially one paragraph of content may look nicer than using lots o space on the page for just two sentences.

Reply:
p > 2p

Reply:
My reasoning is think it's nice to keep as much space on the page for the list of tools as possible. It looks like an improvement to me now!